apps/dashboard/src/hooks/use-form-autosave.ts

```diff
@@ -17,12 +17,12 @@ export function useFormAutosave<U extends Record<string, unknown>, T extends Fie
   form: UseFormReturn<T>;
   isReadOnly?: boolean;
   shouldClientValidate?: boolean;
-  save: (data: U) => void;
+  save: (data: U, options: { onSuccess?: () => void }) => void;
```

Comment:
allows to listen to the successfull workflow updates and do something in the callback, for example close modal